src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <img
+        src="styles/images/imac.jpeg"
+        alt=""
+        class="card__image"
+      >
+
+      <div class="card__item title title--matgin_top">
+        Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+      </div>
+
+      <div class="card__item code code--margin_top">
+        Код товара: 195434
+      </div>
+
+      <div class="card__item review">
+        <div class="stars stars--4">
```

Comment:
```suggestion
      <div class="card__item review">
        <div class="stars stars--4">
```
не соответствует BEM. stars - блок, но не является элементом внешнего блока